shipgen/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ ${CMAKE_SOURCE_DIR}/generators
 ${CMAKE_SOURCE_DIR}/shipdata
 ${CMAKE_SOURCE_DIR}/veto
 ${genfit2_INCDIR}
+${TPYTHIA6_INCLUDE_DIR}
```

Comment:
Can we define a CMake Target as a dependency instead of manually including the libraries?

Reply:
This can be addressed later.